tools/export_to_path/export_to_path.py

```diff
@@ -15,38 +14,47 @@
 parser = optparse.OptionParser()
 parser.add_option('-d', '--export_dir', help='Directory where to export the datasets')
 (options, args) = parser.parse_args()
-if not options.export_dir:
-    parser.error('Export directory cannot be empty')
-
-dir_prefix = "/mnt/sally"
-dir_suffix = 'mzML_profile'
-full_export_dir = os.path.join(dir_prefix, options.export_dir.lstrip(os.sep), dir_suffix)
-real_export_dir = os.path.realpath(full_export_dir)
-if not real_export_dir.startswith(dir_prefix):
-    raise Exception("'%s' must be a subdirectory of '%s'" % (real_export_dir, dir_prefix))
-if not os.path.exists(real_export_dir):
-    os.makedirs(real_export_dir)
+
+
+def get_path():
+    dir_prefix = '/mnt/sally'
+    dir_suffix = 'mzML_profile'
+    if not options.export_dir:
```

Comment:
I don't see the need for this to be moved inside the function. Especially since its name is `get_path`.